scripts/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "moduleResolution": "Node"
+  },
```

Comment:
nit question: Should we make these options closer to our other tsconfigs? Like https://github.com/gadget-inc/js-clients/blob/98f6b02e00cb6a1624a5ed81cb949fea5a0b6519/packages/api-client-core/tsconfig.json#L3-L11

super nit: not sure if it's case-sensitive, but we typically use the all lowercase value `"node"`

Reply:
I added the `lib` and `type` options but *I don't think* the other options referenced in `api-client-core` are relevant to the script folder.

I also lowercased `"node"`. I don't have a preference, it was my editor that autocompleted `"Node"`.